### DISCUSSION TITLE
Add mpv microsoft store version path detection to syncplay

### DISCUSSION THREAD

**Discussion by soredake** [Syncplay/syncplay#608] [Ideas]
https://github.com/Syncplay/syncplay/commit/bed49869795f5bffd59d3dad657a965c2644b107

MPV microsoft store version: https://apps.microsoft.com/detail/9P3JFR0CLLL6

MPV microsoft store version path: `%LOCALAPPDATA%\Microsoft\WindowsApps\mpv.exe`

**Comment by Et0h** [comment]
I see you have now proposed this in https://github.com/Syncplay/syncplay/pull/662 so I'll close this discussion to avoid duplication.
